### PR TITLE
Fixes monkeys eating your organs while still inside you

### LINKED ISSUE #80367: Flaky test monkey_business: Cannot modify null.tongue.
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/7226916128/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
monkey_business: [20:58:21] Runtime in code/modules/surgery/bodyparts/dismemberment.dm,213: Cannot modify null.tongue.
proc name: on bodypart remove (/obj/item/organ/internal/tongue/on_bodypart_remove)
src: the tongue (/obj/item/organ/internal/tongue)
src.loc: null
call stack:
the tongue (/obj/item/organ/internal/tongue): on bodypart remove(null)
the tongue (/obj/item/organ/internal/tongue): bodypart remove(null, null, null)
the tongue (/obj/item/organ/internal/tongue): forced removal(the tongue (/obj/item/organ/internal/tongue), John Doe\'s head (/obj/item/bodypart/head), 0, 1, null, 1)
the tongue (/obj/item/organ/internal/tongue):  SendSignal("movable_moved", /list (/list))
the tongue (/obj/item/organ/internal/tongue): Moved(John Doe\'s head (/obj/item/bodypart/head), 0, 1, null, 1)
the tongue (/obj/item/organ/internal/tongue): doMove(John Doe (283) (/mob/living/carbon/human/consistent))
the tongue (/obj/item/organ/internal/tongue): doMove(John Doe (283) (/mob/living/carbon/human/consistent))
the tongue (/obj/item/organ/internal/tongue): forceMove(John Doe (283) (/mob/living/carbon/human/consistent))
John Doe (283) (/mob/living/carbon/human/consistent): put in hand(the tongue (/obj/item/organ/internal/tongue), 2, 0, 1)
John Doe (283) (/mob/living/carbon/human/consistent): put in hands(the tongue (/obj/item/organ/internal/tongue), 0, 1, 0, 1)
/datum/ai_behavior/consume (/datum/ai_behavior/consume): perform(2, /datum/ai_controller/monkey (/datum/ai_controller/monkey), "bb_food_target", "BB_NEXT_HUNGRY")
/datum/ai_controller/monkey (/datum/ai_controller/monkey): ProcessBehavior(2, /datum/ai_behavior/consume (/datum/ai_behavior/consume))
/datum/ai_controller/monkey (/datum/ai_controller/monkey): process(0.1)
AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): ignite(0)
Master (/datum/controller/master): RunQueue()
Master (/datum/controller/master): Loop(2)
Master (/datum/controller/master): StartProcessing(0) at code/modules/surgery/bodyparts/dismemberment.dm:213
```

### PR DESCRIPTION
**Pull request by Time-Green**
Fixes #80367, #80372 (hopefully)

:cl:
fix: Monkeys will no longer eat your organs while they're still inside of you
/:cl:

### CODE REVIEW

File: code/modules/mob/inventory.dm [hunk 1]
```diff
@@ -186,9 +186,9 @@
 /mob/proc/put_in_hand_check(obj/item/I)
 	return FALSE //nonliving mobs don't have hands
 
-/mob/living/put_in_hand_check(obj/item/I)
-	if(istype(I) && ((mobility_flags & MOBILITY_PICKUP) || (I.item_flags & ABSTRACT)) \
```
Comment (Time-Green) [inline code/modules/mob/inventory.dm:190]: This feels like poking at a wasps nest, but this seems like a mistake that allows for the pick-up of abstract objects

**Comment by Time-Green** [comment]
Different approach now. We ANCHOR the organs inside the mob, thank @Fikou for the great suggestion!!

**Comment by Fikou** [comment]
> Different approach now. We ANCHOR the organs inside the mob, thank @Fikou for the great suggestion!!

NO PLEASE MAN NO DONT

**Comment by Time-Green** [comment]
yeah okay

**Review by Ghommie — APPROVED** [review]
Damn, that's savage.

**Pull request merged**